extensions/analyticsdx-vscode-core/src/commands/openStudio.ts

```diff
@@ -19,45 +19,32 @@ import { AppGatherer, AppMetadata } from './gatherers/appGatherer';
 
 export function baseStudioPath() {
   return (
-    vscode.workspace
-      .getConfiguration()
-      .get<string>('analyticsdx-vscode-core.studio.path')
-      ?.trimLeft() || '/analytics/'
+    vscode.workspace.getConfiguration().get<string>('analyticsdx-vscode-core.studio.path')?.trimStart() || '/analytics/'
   );
 }
 
-// REVIEWME: just get the url from sfdx and open in a vscode WebViewPanel?
-
 // This will have sfdx print the url, which this will read and use with vscode.env.openExternal() so that
 // it works in Code Builder (and Github Codespaces).
 
-class OpenStudioExecutor<T> extends SfdxCommandletExecutorWithOutput<T> {
+class OpenOrgPathExecutor<T> extends SfdxCommandletExecutorWithOutput<T> {
```

Comment:
New Data Manager url isn't under `/analytics` so I had to refactor this a bit to support that.